mesa/experimental/jupyter_viz.py

```diff
@@ -51,6 +51,23 @@ def make_model():
         set_current_step(0)
         return model
 
+    def calculate_space_size():
+        if model_parameters["N"] > 50:
```

Comment:
Once again, the grid size is independent of the number of agents. Having few agents and having many agents should result in the same marker size and figure size.

Reply:
Yes, agents less than 50 the figure size of default used by matplotlib. I start to scale the it more num of agents. 

Ideal would be scale the image size based on screen size, but this doesn't seem feasible. Any other ideas?

Reply:
> I start to scale the it more num of agents.

Number of agents shouldn't have effect on image size for the grid.